monasca-aggregator/Dockerfile

```diff
@@ -1,27 +1,47 @@
-from alpine:3.7
+from golang:1.10.3-alpine3.7 as builder
 
 arg AGGREGATOR_REPO=https://github.com/monasca/monasca-aggregator
 arg AGGREGATOR_BRANCH=master
 
 env GOPATH=/go
 
-run apk add --no-cache openssl py2-jinja2 librdkafka && \
-  apk add --no-cache --virtual build-dep \
-    librdkafka-dev git go glide make g++ openssl-dev && \
-  mkdir -p $GOPATH/src/github.com/monasca/monasca-aggregator && \
+run apk add --no-cache openssl py2-jinja2 && \
+  apk add --no-cache --virtual build-dep git go glide make g++ openssl-dev
```

Comment:
Please, remove installing `go` from here. It's installing `go` in version `1.9.4`.
`(17/25) Installing go (1.9.4-r0)`

Reply:
Interesting, even after removing go from the list, it installs as a dependency of glide